usagov/usagov-2021
# Usagov 1286 add ceo text

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1286

## Description

Added text to the bottom of the English and Spanish CEO form and added text to the results page accordions
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Test CEO Page text addition
- Goto http://localhost/elected-officials and you should see text added after the form
- 
<img width="1512" alt="Screenshot 2024-01-05 at 2 27 45 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/9872dfe7-a37f-464c-84d2-1c61bb1254be">
Goto the CEO Spanish page http://localhost/es/funcionarios-electos and you should see the text after the form
- Test Results Page Text Addition
- Goto http://localhost/elected-officials and fill out the form and submit to goto the results page.  You should see text added to the accordion
<img width="1512" alt="Screenshot 2024-01-05 at 2 30 50 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/cfc01e75-cdfe-41ac-92b6-796871269e72">
Goto the CEO Spanish page http://localhost/es/funcionarios-electos fill out the form and submit to see the add accordion text

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
